site/components/ViewAllContents.tsx

```diff
@@ -476,6 +476,7 @@ const ViewAllContents = ({
 }: ViewAllContentProps): ReactElement => {
     const [selectedServices, setSelectedServices] = useState<Service[]>([]);
     const [selectedOperators, setSelectedOperators] = useState<ConsequenceOperators[]>([]);
+    const [servicesDataSource, setServicesDataSource] = useState<Datasource | undefined>();
```

Comment:
```suggestion
    const [servicesDataSource, setServicesDataSource] = useState<Datasource | "all">("all");
```

If the dropdown is setting this to a value of "all" then we should ideally include it in the type here and default to it

Reply:
done